src/core/column-type/column.model.d.ts

```diff
@@ -0,0 +1,32 @@
+import {IEditorOptions} from "./email.column";
```

Comment:
please align declarations, now indents are different - somewhere spaces, somewhere tabs. In the project we uses tabs by convention now